systemtests/nirelaydriver.pinmap

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PinMap schemaVersion="1.5" xmlns="http://www.ni.com/TestStand/SemiconductorModule/PinMap.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<Instruments>
+		<NIRelayDriverModule name="RelayDriver1" numberOfControlLines="64" />
+		<NIRelayDriverModule name="RelayDriver2" numberOfControlLines="64" />
+	</Instruments>
+	<Pins></Pins>
+	<PinGroups></PinGroups>
+	<Relays>
+		<SiteRelay name="SiteRelay1" openStateDisplayLabel="Open" closedStateDisplayLabel="Closed" />
+		<SiteRelay name="SiteRelay2" openStateDisplayLabel="Open" closedStateDisplayLabel="Closed" />
+		<SystemRelay name="SystemRelay1" openStateDisplayLabel="Open" closedStateDisplayLabel="Closed" />
+	</Relays>
+	<RelayGroups>
+		<RelayGroup name="RelayGroup1">
+			<RelayReference relay="SiteRelay1" />
+			<RelayReference relay="SiteRelay2" />
+			<RelayReference relay="SystemRelay1" />
+		</RelayGroup>
+	</RelayGroups>
+	<RelayConfigurations>
+		<RelayConfiguration name="RelayConfiguration1">
+			<RelayPosition relay="SiteRelay1" position="Closed" />
+			<RelayPosition relay="SiteRelay2" position="Closed" />
+			<RelayPosition relay="SystemRelay1" position="Open" />
+		</RelayConfiguration>
+	</RelayConfigurations>
```

Comment:
I recommend omitting this section because it doesn't appear to be used. This section would be necessary to test the apply_relay_configuration method, but that method is tested in a [unit test.](https://github.com/ni/nitsm-python/blob/90345243cf80d46043c4541e414f607af3c2f3f7/tests/test_nirelaydriver.py#L134)